modules/targetanalysis/src/target_loader.cpp

```diff
@@ -29,7 +29,7 @@
     SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
 
-#include "json/json.h"
+//#include "json/json.h"
```

Comment:
No longer necessary, see [this commit](https://github.com/UWARG/computer-vision/commit/d499b43e557a80703e8843e04d66c71875ecf539). JSONCpp should be recognized by CMake now.